packages/module/src/InvalidObject/InvalidObject.test.js

```diff
@@ -0,0 +1,11 @@
+import InvalidObject from './InvalidObject';
+import React from 'react';
+import { mount } from 'enzyme';
```

Comment:
Update to use react testing library.  If you need any help feel free to reach out.